spec/workers/send_eoc_deadline_reminder_email_to_candidates_worker_spec.rb

```diff
@@ -2,232 +2,111 @@
 
 RSpec.describe SendEocDeadlineReminderEmailToCandidatesWorker, :sidekiq do
   describe '#perform' do
-    context 'when the candidate is in Apply 1' do
-      it 'returns an application when the deadline is 2 months away' do
-        travel_temporarily_to(CycleTimetable.apply_1_deadline_first_reminder) do
-          candidate = create(:candidate)
+    it 'returns an application when the deadline is 2 months away' do
+      travel_temporarily_to(CycleTimetable.apply_deadline_first_reminder) do
+        candidate = create(:candidate)
 
-          create(
-            :application_form,
-            candidate:,
-            phase: 'apply_1',
-            application_choices: [create(:application_choice, :application_not_sent)],
-            recruitment_cycle_year: RecruitmentCycle.current_year,
-          )
-
-          described_class.new.perform
-
-          email_for_candidate = email_for_candidate(candidate)
-
-          expect(email_for_candidate).to be_present
-        end
-      end
-
-      it 'does not return an application where the candidate is unsubscribed' do
-        allow(CycleTimetable).to receive(:need_to_send_deadline_reminder?).and_return(:apply_1)
-
-        unsubscribed_candidate = create(:candidate, unsubscribed_from_emails: true)
-        create(:application_form, candidate: unsubscribed_candidate)
+        create(
+          :application_form,
+          candidate:,
+          application_choices: [create(:application_choice, :application_not_sent)],
+          recruitment_cycle_year: RecruitmentCycle.current_year,
+        )
```

Comment:
Just confirming if is correct:

> CycleTimetable.apply_deadline_first_reminder
=>  Mon, 22 Jul 2024 00:00:00.000000000 BST +01:00

> CycleTimetable.apply_deadline_second_reminder
=>  Mon, 19 Aug 2024 00:00:00.000000000 BST +01:00

Is this expected and aligned in relation to the dates [the thread](https://ukgovernmentdfe.slack.com/archives/C077NRS8K8D/p1718899158123639?thread_ts=1718193462.058709&cid=C077NRS8K8D)? Should be the beginning of the week?